docs/current/sdk/nodejs/guides/114934-nodejs-ci.md

```diff
@@ -0,0 +1,55 @@
+---
+slug: /114934/nodejs-ci
+displayed_sidebar: "current"
+---
+
+# Dagger Node.js SDK in CI
+
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem';
+
+<Tabs defaultValue="github-actions"
+groupId="ci-environment"
+values={[
+{label: 'GitHub Actions', value: 'github-actions'},
+{label: 'CircleCI', value: 'circleci'},
+{label: 'GitLab', value: 'gitlab'},
+{label: 'Jenkins', value: 'jenkins'},
+]}>
+
+<TabItem value="github-actions">
+
+```yaml title=".github/workflows/dagger.yaml" file=../snippets/nodejs-ci/actions.yml
+```
+
+Ensure that your `package.json` contains `@dagger.io/dagger` which can be installed using the [documentation](https://docs.dagger.io/sdk/nodejs/835948/install).
```

Comment:
```suggestion
Ensure that your `package.json` contains `@dagger.io/dagger`, as described in the [SDK installation documentation](https://docs.dagger.io/sdk/nodejs/835948/install).
```

More descriptive link for SEO

Also could you please update the link to be a relative link?

Same comments as above for other tabs on this page.